clean.py

```diff
@@ -0,0 +1,84 @@
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import StandardScaler
+from sklearn.preprocessing import MinMaxScaler
+
+""" This module is meant for data cleaning """
+
+def normalize_overall_min_max(df_train, df_test, target):
+    ''' This function takes the train and test dataframes and 
```

Comment:
Doc strings can be added after the module is complete, but I think the format is a single line brief description, then a empty line, then a longer description, then a description of inputs, and then outputs.